apps/dot-voting/app/Decisions.js

```diff
@@ -155,6 +166,7 @@ const Decisions = ({ decorateVote }) => {
           app={app}
           vote={currentVote}
           userAccount={connectedAccount}
+          tokenContract={tokenContract}
```

Comment:
Looks like this is the only place that `tokenContract` is used. Given that `VoteDetails` can also pull `tokenAddress` out of `appState` and do all of this logic itself, can we move it all there?